mypy.ini

```diff
@@ -11,3 +11,7 @@ warn_return_any = True
 [mypy-rlpython.*]
 # Avoid error: Skipping analyzing "rlpython": found module but no type hints or library stubs
 ignore_missing_imports = True
+
+[mypy-pytest.*]
+# Avoid error: Skipping analyzing "rlpython": found module but no type hints or library stubs
+ignore_missing_imports = True
```

Comment:
Let's remove this config and istead add `pytest` in deps of `[testenv:lint]`

Reply:
Wow. I've found that the same works with `rlpython`. We can remove its config as well.